build-system/test-server.js

```diff
@@ -111,5 +111,17 @@ app.use('/form/post', function(req, res) {
   });
 });
 
+/**
+ * Serve entry point script url
+ */
+app.get('/dist/ww.js', function(req, res, next) {
```

Comment:
In your test browser, can you try to load "http://localhost:9876/dist/ww.js"? It worked on our end.